alvr/server_openvr/src/lib.rs

```diff
@@ -414,30 +409,57 @@ pub extern "C" fn shutdown_driver() {
     SERVER_CORE_CONTEXT.write().take();
 }
 
+// Check that there is no active dashboard instance not part of this driver installation
+pub fn should_initialize_driver(driver_layout: &afs::Layout) -> bool {
+    if let Some(proc) = sysinfo::System::new_all()
+        .processes_by_name(OsStr::new(&afs::dashboard_fname()))
+        .next()
+    {
+        if let Some(dashboard_path) = proc.exe() {
+            return driver_layout.dashboard_exe() == dashboard_path;
+        }
+    }
+
+    true
+}
```

Comment:
Since this will pick up all dashboards w/o ordering another dashboard might still be launched but missed by this logic. Pretty sure this has to actually check all matching, and just continue except if there's a mismatch and then return false.

Reply:
Yeah

Reply:
Actually, since we would kill any other dashboards by the other piece of code, this shouldn't happen. but you're right in the sense that an older version of ALVR might not have the dashboard killing logic

Reply:
If we have the case where an old driver and a new driver are launched, the new driver should quit because that's the only one with the quitting logic at launch. This means that if a separate dashboard (possibly belonging to an old server) is found, the new driver should quit.
Ideally, if the new driver finds its own dashboard, we would like to kill any other driver. But this is not possible unfortunately because they are tied to the same SteamVR process. It would be possible to maybe delegate the dashboard at unregistering the other drivers, but this might become unnecessarily complicated

Reply:
In the end this is a best effort check. But should work alright for concurrent installations of ALVR strictly greater than v20.13.0